controller/main.go

```diff
@@ -98,12 +104,12 @@ func main() {
 	setupErr, testErr := controller.RunTest(*testNameArg)
 	if setupErr != nil {
 		logrus.Errorf("Test %v encountered an error during setup (test did not run):", *testNameArg)
-		logrus.Error(setupErr)
+		fmt.Fprintln(logrus.StandardLogger().Out, setupErr)
```

Comment:
this lil trick here shows up in a lot of places in both kurtosis and ave-e2e-tests. what do you think of us just writing a wrapper around logrus like kurtosislogger and then providing an api that pretty prints these so that test writers dont have to worry about it?

Reply:
I'm not necessarily opposed to it, though we actually rely on some deeper functionality in logrus, particularly in Kurtosis - e.g. spawning new loggers, forcing the colors, and changing the output destinatinons. I think if we were to do it, we'd need to poke a lot of function calls through. If we did go forward with it, I def think it should be a separate PR.

Reply:
i'll make an issue for it 👍  this is good to merge